core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html

```diff
@@ -0,0 +1,24 @@
+<form
+    id="ai-text-prompt"
+    *ngIf="form"
+    [formGroup]="form"
+    (ngSubmit)="onSubmit()"
+    autocomplete="off"
+>
+    <div *ngFor="let control of dynamicControls">
+        <ng-container>
+            <span class="p-input-icon-right">
+                <input
+                    #input
+                    [formControlName]="control.key"
+                    type="text"
+                    pInputText
+                    placeholder="Ask AI to write something"
+                />
+                <button type="submit">
+                    <i class="pi pi-send"></i>
+                </button>
```

Comment:
We can use a primeNG button
```suggestion
                    <p-button icon="pi pi-send" styleClass="p-button-rounded p-button-text" type="submit">
        </p-button>
```